src/flags.ts

```diff
@@ -1,3 +1,4 @@
 export const Flags = {
-  HELP_MODAL: false
+  HELP_MODAL: false,
+  AUTO_CHECK_UPDATES: true
```

Comment:
why add a flag which is set to true anyway?

Reply:
Yea, I can remove this for now. Was preparing for the use case of SA being used in a self-service kiosk of sorts 🙃